react/features/base/buttons/CopyButton.js

```diff
@@ -41,11 +42,12 @@ const styles = theme => {
             }
         },
         content: {
+            display: 'flex',
             overflow: 'hidden',
             textOverflow: 'ellipsis',
             whiteSpace: 'nowrap',
             maxWidth: 292,
-            marginRight: 16,
+            marginRight: BaseTheme.spacing[3],
```

Comment:
don't think we want to import BaseTheme, think we need to take the spacing from `theme.spacing`

Reply:
Ah, yes. You're right. Updated.